oauthproxy_test.go

```diff
@@ -1748,6 +1748,9 @@ func TestRequestSignature(t *testing.T) {
 
 func TestGetRedirect(t *testing.T) {
 	opts := baseTestOptions()
+	opts.ReverseProxy = true
+	opts.WhitelistDomains = append(opts.WhitelistDomains, ".example.com")
+	opts.WhitelistDomains = append(opts.WhitelistDomains, ".example.com:8443")
```

Comment:
I don't think this line is needed anymore, right?

Reply:
It still has to be there, sadly.